mmv1/products/dataplex/Glossary.yaml

```diff
@@ -0,0 +1,121 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Glossary'
+min_version: beta
+description: |
+  Represents a collection of categories and terms defined by the user. Glossary is a top level resource and is the GCP parent resource of all the categories and terms within it.
+docs:
+base_url: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/glossaries?glossary_id={{glossary_id}}'
+update_verb: 'PATCH'
+update_mask: true
+timeouts:
+  insert_minutes: 5
+  update_minutes: 5
+  delete_minutes: 5
+autogen_async: true
+import_format: ['projects/{{project}}/locations/{{location}}/glossaries/{{glossary_id}}']
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+iam_policy:
+  exclude_import_test: true
```

Comment:
The import tests were likely failing because the primary_resource_names were incorrect.

```suggestion
```